runtests.py

```diff
@@ -41,6 +54,9 @@
         "ppmt",
         "pv",
         "rate",
+        # More standard names should be preferred
+        "alltrue",  # all
+        "sometrue",  # any
```

Comment:
For ``alltrue`` can't we just use ``alltrue = all`` and call it a day?
Considering it's supposed to be public function (correct me if I'm wrong here) I feel it would not be wise to just skip it.

Reply:
> Considering it's supposed to be public function

It's not in the refguide, and the general opinion seems to be "should be deprecated/removed" (see https://github.com/numpy/numpy/issues/14584), so I think it's best to exclude it.